scale/job/configuration/configurators.py

```diff
@@ -263,15 +263,15 @@ def configure_scheduled_job(self, job_exe, job_type, interface):
             ScheduledExecutionConfigurator._configure_regular_job(config, job_exe, job_type)
 
         # Configure items that apply to all tasks
-        self._configure_all_tasks(config, job_exe, job_type)
+        self._configure_all_tasks(config, job_exe, job_type, debug_level)
 
         # Configure secrets
         config_with_secrets = self._configure_secrets(config, job_exe, job_type, interface)
 
         job_exe.configuration = config.get_dict()
         return config_with_secrets
 
-    def _configure_all_tasks(self, config, job_exe, job_type):
+    def _configure_all_tasks(self, config, job_exe, job_type, debug_level):
```

Comment:
I would rename `debug_level` to `logging_level`. Also be sure to add the new parameter to the docstring.